scan-indeed.py

```
@@ -0,0 +1,275 @@
+        if not title_filter(job["title"]):
+            skipped_title += 1
+            # Record in history as skipped
+            if not args.dry_run:
+                with open(SCAN_HISTORY_PATH, "a") as f:
+                    f.write(f"{job['url']}\t{today}\tindeed-ie\t{job['title']}\t{job['company']}\tskipped_title\n")
+            continue
+        new_jobs.append(job)
+
+    # Summary
+    print(f"\n{'━' * 50}")
+    print(f"📊 Results:")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)